.github/workflows/pypi.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish vembrane to PyPI
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-and-publish:
+    name: build and publish python dists to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```

Comment:
Since you build `wheel`s: Is it intentional to only build for 3.7 and not 3.8?

Reply:
tbh I have no clue what I'm doing w.r.t. python packaging

Reply:
> I have no clue what I'm doing w.r.t. python packaging

I can't remember ever hearing anyone say the opposite about themselves! I'm pretty sure we're all in the same boat even if some tell differently XD

Reply:
I think we can restrict poetry to build either sdist or wheel only via the `--format` option, if that helps?

Reply:
Or do we have to include multiple python versions in this step?

Reply:
Nvm, sorry, this builds universal wheels and as such doesn't need multiple runs for different versions.